EIPS/eip-7870.md

```diff
@@ -0,0 +1,152 @@
+author: Parithosh Jayanthi (@parithosh), Kevaundray Wedderburn (@kevaundray), Josh Rudolf (@jrudolf), Dankrad Feist (@dankrad), Justin Traglia (@jtraglia), Ignacio Hagopian (@jsign), George Kadianakis (@asn-d6)
+discussions-to: https://ethereum-magicians.org/t/hardware-and-bandwidth-recommendations-for-full-nodes-and-validators/22675
+status: Draft
+type: Informational
+created: 2025-01-26
+---
+
+## Abstract
+
+This proposal specifies hardware and bandwidth recommendations for full nodes and validators. Validators are split into attesters and proposers. The resource-intensive aspect of block proposal lies in creating the block and broadcasting the data required for attesters to validate it. We use the term "local block builder" to further emphasize this separation.
+
+We note that you may be able to run a client with less than the recommended specifications, however benchmarks and decision-making will be made with respect to these recommendations.
+
+## Motivation
+
+Clear system specifications are crucial for:
+
+- Ensuring meaningful benchmark comparisons across different implementations
+- Enabling informed decision-making about protocol upgrades and their resource usage implications
+- Providing clear guidance for node operators with respect to the future
+
+Without a shared understanding of target hardware specifications:
+
+- Benchmark results lose significance due to inconsistent testing environments
+- Decision-making becomes challenging for implementation choices, as performance characteristics are heavily hardware-dependent
+- Network participants lack clear guidance for hardware investments
+
+## Specification
+
+### Recommended hardware and bandwidth specifications
+
+| Node type | Storage | RAM | CPU Cores | CPU Single Thread/Multithread rating | Download/Upload speed |
+| -------- | -------- | -------- | --------| -------- |--------|
+| Full node     | 4TB     | 32GB    | 4 cores/8 threads  | 1000 / 3000 | 50 Mbps / 15 Mbps |
+| Attester      | 4TB     | 64GB    | 8 cores/16 threads | 3500 / 25000 | 50 Mbps / 25 Mbps |
+| Local block builder | 4TB     | 64GB    | 8 cores/16 threads | 3500 / 25000 |  100 Mbps / 50 Mbps |
+
+- The CPU rating is a measure of how powerful a particular CPU is for single and multithreaded respectively. All numbers referenced are normalized according to PassMark.
+
+### Recommended Prebuilds (Attester and Local block builder)
+
+#### Minisforum UM790 Pro
+
+We recommend the Minisforum UM790 Pro with the following modifications:
+
+- Minisforum UM790 Pro
+- 8 cores
+- 16 threads
+- The barebones model will need to be updated to include 32GB RAM and 4TB storage
+- CPU Rating
+  - Single Thread: 3903
+  - Multithread: 30367
+
+#### ASUS NUC 14 Pro
+
+- 16 cores (6 performance + 8 efficiency cores + 2 Low power efficiency cores)
+- 22 threads
+- CPU Rating
+  - Single Thread: 3520
+  - Multithread: 25158
+
+### Recommended Prebuild (Full node)
```

Comment:
To match the order of the recommended hardware table, the full node section should be first.

Reply:
Since we don't want brands, I'll remove this whole section on recommended prebuilds